pymc3/examples/test_DiscreteUniform.py

```diff
@@ -0,0 +1,38 @@
+from numpy import array
```

Comment:
This should also be in the `tests` directory and probably just added to an existing test file there, like a test_step_methods or something.

Reply:
Makes sense. I have now done this.

Reply:
I still these files committed here.